o11y/metrics/datadog/datadog.go

```diff
@@ -252,6 +252,11 @@ func (d Sink) MetricDisruptionScheduled(tags []string) error {
 	return d.metricWithStatus(d.prefix+"schedule.disruption_scheduled", tags)
 }
 
+// MetricPausedCron reports when a disruption cron has reconciled in a paused state
+func (d Sink) MetricPausedCron(tags []string) error {
+	return d.metricWithStatus(d.prefix+"schedule.paused", tags)
```

Comment:
The metric name doesnt exactly match the function name, but I wanted to get closer to the prefix for the other cron metrics.